### ISSUE TITLE
large numeric values appearing as NaN

### ISSUE THREAD

**Issue by unek** [jakob/Postico#585]
huuuge `numeric` type values appear as `NaN` in the content/results view. not entirely sure it's a postico issue though but looks like it

![image](https://user-images.githubusercontent.com/1358551/44603652-a310c480-a7e3-11e8-81f1-8e88cae6110c.png)

Postico version:
`Version 1.4.2 (2845)`

PostgreSQL version:
`PostgreSQL 10.3 on x86_64-apple-darwin14.5.0, compiled by Apple LLVM version 7.0.0 (clang-700.1.76), 64-bit`

**Comment by jakob** [comment]
Yes, this is a Postico issue. Postico tries to read `NUMERIC` values as `NSDecimalNumber`, but PostgreSQL supports more digits than Cocoa.

I'm not sure how to best fix this. Maybe we should just display every value like text (like PostgreSQL sends it)

**Comment by jakob** [comment]
Just curious: Did you discover this while testing, or do giant numbers like this actually come up in your field of work?

**Comment by unek** [comment]
i was just testing the bignumber implementation of my app and got a little carried away haha
